packages/admin-panel/src/pages/resources/AccessRequestsPage.js

```diff
@@ -84,6 +84,7 @@ const USER_COLUMNS = [
     type: 'bulkEdit',
     width: 150,
     actionConfig: {
+      title: 'Edit & Approve Access Request',
```

Comment:
Your new solution allows us to get more specific here too!
```suggestion
      title: 'Edit & Approve Access Requests',
```